index.html

```diff
@@ -0,0 +1 @@
+<meta http-equiv="refresh" content="0; url=https://raw.githubusercontent.com/molstar/molstar-volseg/molstar-volseg-extension/molstar-extension/site/index.html">
```

Comment:
Why is this page here?

Reply:
@dsehnal, initially I tried to deploy it from github UI, which allows to deploy gh pages only from root or docs